### ISSUE TITLE
Issuses with reverse proxy

### ISSUE THREAD

**Issue by Crowley007** [theotherp/nzbhydra2#199]
I just migrated to Hydra 2, and it seems to be working  great. Thank you for the software!

This is not a Hydra problem really so no log, but everything else works so it must be doing something differently.

No matter what I try, I can't get reverse proxy to work with Hydra 2 and Nginx. To be fair, I can't really use it and have been relying on guides and testing what works. 

This line worked with Nzbhydra 1, but wont work anymore. 

"location /nzbhydra {
		proxy_pass http://192.168.0.2:5075/nzbhydra/;"
}

I tried the new settings from the wiki, but it just wont work through nginx. Accessing http://192.168.0.2:5075/nzbhydra/ works fine. 

Config is attached, I hope someone could take a look and inform me what i'm doing wrong.

[nginx.txt](https://github.com/theotherp/nzbhydra2/files/2101652/nginx.txt)
![hydra settings](https://user-images.githubusercontent.com/23026322/41403783-6596dede-6fce-11e8-9f31-97093a0fc268.PNG)

**Comment by theotherp** [comment]
See
https://github.com/theotherp/nzbhydra2/wiki/Exposing-Hydra-to-the-internet-and-using-reverse-proxies

**Comment by Crowley007** [comment]
I did, and as my config shows I copied the lines from that guide and only changed the IP and url base to match Hydra settings. But it just wont work.

One weird side effect that might be related.  I have to use "http://192.168.0.2:5075" in Sonarr and Radarr, if I add the url base "/nzbhydra" it wont connect.

**Comment by theotherp** [comment]
Sorry, I only read the issue in my mails and didn't see the attachment.

Your nginx.conf says `location /nzbhydra/ {` but it should be `location /nzbhydra {`. Note the trailing slash (or lack thereof). So change that in your config.

You didn't post the debug infos ZIP as requested by the issue template so I can't take a look at your configuration or log file. There is no way that `http://192.168.0.2:5075` could work for Sonarr or Radarr if everything is configured as it should.

**Comment by Crowley007** [comment]
Since Hydra works, I assumed the problem is with my ngingx config. That Slash makes no difference either way so perhaps there is something else going on.

I have now added the hydra log.

[nzbhydra-debuginfos-2018-06-15-12-52.zip](https://github.com/theotherp/nzbhydra2/files/2105602/nzbhydra-debuginfos-2018-06-15-12-52.zip)

**Comment by theotherp** [comment]
Perhaps a dump question, but did you restart Hydra after making the changes for the URL base?

**Comment by Crowley007** [comment]
Not a dumb question at all. Hydra 2 seemed to get that url base from Hydra1, and I did restart a few times since I installed it as a service after.

But now some combination of restarts and ngingx config changes did fix it and all is working as expected, both reverse proxy and URL base in Sonarr/radarr.

I guess this was a pretty pointless issue, but nevertheless, thank you for the support and great software!
